paddlenlp/transformers/tokenizer_utils.py

```diff
@@ -1363,7 +1363,13 @@ def get_offset_mapping(self, text):
             if token in self.all_special_tokens:
                 token = token.lower() if hasattr(
                     self, "do_lower_case") and self.do_lower_case else token
-            start = text[offset:].index(token) + offset
+            # Deal with special greek letter with 2 forms (sigma)
```

Comment:
可以加个特殊处理的说明，例如：
“The greek letter "sigma" has 2 types of lowercase. When used at the end of a [letter-case](https://en.wikipedia.org/wiki/Letter_case) word (one that does not use [all caps](https://en.wikipedia.org/wiki/All_caps)), the [final form](https://en.wikipedia.org/wiki/Final_form) (ς) is used. Otherwise, the form (σ) is used”

Reply:
Done